client/src/components/GoogleMaps.vue

```diff
@@ -310,12 +330,55 @@ export default Vue.extend({
   width: 70%;
 }
 
+.control-button__streak {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+
+  &.no-streak {
+    filter: opacity(0.1);
+  }
+}
+
+.control-button__streak__count {
+  position: absolute;
+  bottom: 0.25em;
+  font-family: "Syne", sans-serif;
+  font-size: 200%;
+  line-height: 1;
+  color: $primary-color;
+  text-shadow: -1px -1px 1px #24242485;
+}
+
+.test {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+}
+
```

Comment:
what is `.test`?

Reply:
good catch, it was for test purposes only, it has now been removed.